src/config/config.js

```diff
@@ -3,6 +3,7 @@ const Config = {
     "serverUrl": process.env.REACT_APP_SERVER_URL || "http://localhost:3003",
     "infuraKey": process.env.REACT_APP_INFURA_KEY,
     "networkId": process.env.REACT_APP_NETWORK_ID || 42,
+    "recaptcha": "6LeOaJIUAAAAAKB3DlmijMPfX2CBYsve3T2MwlTd",
```

Comment:
does it has to be here? it's just a testing key?

Reply:
hmm its public and wont change.
but i guess you can make change and move it to env